File: Tareas/Ejercicios JS/ScrumMaster/Ejercicio1.js
//Escribe una función que tome un número como argumento y 
//  muestre por consola los números pares desde 0 hasta ese número.

function numPares(n) {
    for (let i = 0; i <= n; i += 2) {   // Recorremos los números de 2 en 2
        console.log(i + " ");
    }
}

let numero = 18;    // Se le asigan un valor a la variable
numPares(numero);

File: Tareas/Ejercicios Python/ScrumMaster/Ejercicio1.py
# Escribe una función que tome un número como argumento y
# muestre por consola los números pares desde 0 hasta ese número.

def numPares(n):
    for i in range(0, n+1, 2):
        print(i, end= ' ')

numero = 18   # Se le asigan un valor a la variable
numPares(numero)